packages/graphql/src/schema/subscriptions/generate-subscription-types.ts

```diff
@@ -37,12 +37,13 @@ export function generateSubscriptionTypes({
 
     const eventTypeEnum = schemaComposer.createEnumTC(EventType);
 
-    nodes.forEach((node) => {
+    const shouldInclude = (node: Node) => !node.exclude?.operations.includes("subscribe");
```

Comment:
This function should probably have a more specific name, such as `shouldIncludeSubscriptionOperation`